Adamant/Modules/Chat/View/Managers/ChatDialogManager.swift

```diff
@@ -201,13 +201,15 @@ private extension ChatDialogManager {
         alert.addAction(
             .init(
                 title: .adamant.chat.reviewNodesList,
-                style: .default,
+                style: .destructive,
```

Comment:
Destructive action is not what actually happens, when we tell the user to review the active node list. User is not destroying anything, why not to stick to the default action instead?